src/main/java/frc/robot/commands/ShooterCommand.java

```diff
@@ -5,48 +5,60 @@
 /* the project.                                                               */
 /*----------------------------------------------------------------------------*/
 
+///** Make sure to change things based on updates to ShooterSubsystem and elsewhere
+
 package frc.robot.commands;
 
 import edu.wpi.first.wpilibj2.command.CommandBase;
-////import frc.robot.subsystems.ShooterSubsystem;
 import frc.robot.RobotContainer;
+import frc.robot.subsystems.ShooterSubsystem;
+import frc.robot.utils.PIDFController;
 
 public class ShooterCommand extends CommandBase {
-  /**
-   * Creates a new ShooterCommand.
-   */
-  public ShooterCommand() {
-    // Use addRequirements() here to declare subsystem dependencies.
-
-  }
-
-  // Called when the command is initially scheduled.
-  @Override
-  public void initialize() {
-
-  }
-
-  // Called every time the scheduler runs while the command is scheduled.
-  @Override
-  public void execute() {
-    if (RobotContainer.getController().getAButton() == true) {
-      // ShooterSubsystem.fire();
-    } else if (RobotContainer.getController().getAButton() == false) {
-      // ShooterSubsystem.stopFire();
+
+    private PIDFController shooterAngle = new PIDFController("shooterAngle", 0.0, 0.0, 0.0, 0.0);
+
+    private ShooterSubsystem m_shooter;
+
+    /**
+     * Creates a new ShooterCommand.
+     */
+    public ShooterCommand(ShooterSubsystem shooter) {
+        // Use addRequirements() here to declare subsystem dependencies.
+        m_shooter = shooter;
+        addRequirements(m_shooter);
+    }
+
+    // Called when the command is initially scheduled.
+    @Override
+    public void initialize() {
+        // Turns on the shooter motor - it should stay on the entire match
+        RobotContainer.getShooterSubsystem().fire(0, 0); /** Replace 0s with proper values **/
```

Comment:
Use `m_shooter`. DO NOT USE Robot Container getter - Delete that method in RobotContainer.java